python/src/uagents/registration.py

```diff
@@ -49,6 +95,29 @@ def coerce_metadata_to_str(
     return out
 
 
+async def use_almanac_api(
```

Comment:
Maybe this can be renamed to indicate that it is sending a post message, `almanac_api_post` for e.g.